### PR TITLE
opening to the browser

### LINKED ISSUE #3: external repo link target should be default browser
The problem
clicking on deepfence github buttons, renders github inside the docker desktop.

Expectation
Upon clicking any of the two buttons, open the GitHub url in the default browser.

image

### PR DESCRIPTION
**Pull request by akashyadav24**
resolves #3

### CODE REVIEW

**Comment by ibreakthecloud** [comment]
@akashyadav24 have you tried running it, is it working?

File: ui/src/Components/links.js [hunk 1]
```diff
@@ -10,16 +10,16 @@ export default function Links() {
...
-                href="https://github.com/deepfence/YaraHunter"
                 target="_blank"
+                href="https://github.com/deepfence/YaraHunter"
                 clickable
             />
             <Chip icon={<GitHub />}
                 label="deepfence/ThreatMapper"
                 variant="outlined"
                 component="a"
+                target="_blank"
```
Comment (ibreakthecloud) [inline ui/src/Components/links.js:21]: this won't work!
We need to do something like this: https://github.com/deepfence/secretscanner-docker-extension/pull/5/files

**Comment by akashyadav24** [comment]
@ibreakthecloud  I have tried to run it but 
I'm not able to run in docker desktop 
but I think it it will work fine

**Comment by ibreakthecloud** [comment]
> @ibreakthecloud I have tried to run it but I'm not able to run in docker desktop but I think it it will work fine

I tested this code, in docker desktop it won't work. This is the suggested method:  https://github.com/deepfence/secretscanner-docker-extension/pull/5/files

**Comment by ibreakthecloud** [comment]
What problem are you facing while running in Docker desktop

**Comment by akashyadav24** [comment]
I don't know how to run in docker desktop

**Comment by ibreakthecloud** [comment]
> I don't know how to run in docker desktop

maybe this could help: https://github.com/deepfence/yarahunter-docker-extension#development

**Pull request closed without merging**